rel/config/examples/retainer.conf.example

```diff
@@ -8,6 +8,9 @@ retainer {
     ## set to false to disable retainer
     enable = true
 
+    ## Message retention time, default is 0 means the message will never expire
```

Comment:
## When Message Expiry Interval is not present in a message, use this config for message retention time, default is 0 means the message will never expire.